DOCS.md

```diff
@@ -15,7 +15,7 @@ Sample of building and publishing an image:
 ```yaml
 steps:
   - name: publish_hello-world
-    image: target/vela-docker:v0.1.0
+    image: target/vela-kaniko:v0.1.0
```

Comment:
Thoughts on updating the image version here?

I know it's likely outside the scope of this PR so we can do it in a separate one but I believe the `v0.1.0` image is pretty old.

Reply:
What about putting it at `lastest` and then adding a note that recommend pinned images.

Like we do here:
https://go-vela.github.io/docs/usage/samples/go_modules/

It would allow us to note have to maintain the version and even could just link them to the images in Docker Hub to see versions.

Reply:
@kneal that works for me 👍